time---> [2020-08-04 21:22:56,996] filename and line ---> {{taskinstance.py:616}} INFO message ----> Dependencies all met for <TaskInstance: store_DAG.create_mysql_table 2020-08-04T21:22:31.721187+00:00 [queued]>
time---> [2020-08-04 21:22:57,012] filename and line ---> {{taskinstance.py:616}} INFO message ----> Dependencies all met for <TaskInstance: store_DAG.create_mysql_table 2020-08-04T21:22:31.721187+00:00 [queued]>
time---> [2020-08-04 21:22:57,012] filename and line ---> {{taskinstance.py:834}} INFO message ----> 
--------------------------------------------------------------------------------
time---> [2020-08-04 21:22:57,013] filename and line ---> {{taskinstance.py:835}} INFO message ----> Starting attempt 2 of 2
time---> [2020-08-04 21:22:57,013] filename and line ---> {{taskinstance.py:836}} INFO message ----> 
--------------------------------------------------------------------------------
time---> [2020-08-04 21:22:57,048] filename and line ---> {{taskinstance.py:855}} INFO message ----> Executing <Task(MySqlOperator): create_mysql_table> on 2020-08-04T21:22:31.721187+00:00
time---> [2020-08-04 21:22:57,049] filename and line ---> {{base_task_runner.py:133}} INFO message ----> Running: ['airflow', 'run', 'store_DAG', 'create_mysql_table', '2020-08-04T21:22:31.721187+00:00', '--job_id', '9', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/store_DAG.py', '--cfg_path', '/tmp/tmp46nn1nb_']
time---> [2020-08-04 21:22:58,998] filename and line ---> {{base_task_runner.py:115}} INFO message ----> Job 9: Subtask create_mysql_table [2020-08-04 21:22:58,998] {{settings.py:213}} INFO - settings.configure_orm(): Using pool settings. pool_size=5, max_overflow=10, pool_recycle=1800, pid=1857
time---> [2020-08-04 21:22:59,019] filename and line ---> {{base_task_runner.py:115}} INFO message ----> Job 9: Subtask create_mysql_table /usr/local/lib/python3.7/site-packages/psycopg2/__init__.py:144: UserWarning: The psycopg2 wheel package will be renamed from release 2.8; in order to keep installing from binary please use "pip install psycopg2-binary" instead. For details see: <http://initd.org/psycopg/docs/install.html#binary-install-from-pypi>.
time---> [2020-08-04 21:22:59,020] filename and line ---> {{base_task_runner.py:115}} INFO message ----> Job 9: Subtask create_mysql_table   """)
time---> [2020-08-04 21:22:59,189] filename and line ---> {{base_task_runner.py:115}} INFO message ----> Job 9: Subtask create_mysql_table [2020-08-04 21:22:59,189] {{__init__.py:51}} INFO - Using executor LocalExecutor
time---> [2020-08-04 21:22:59,638] filename and line ---> {{base_task_runner.py:115}} INFO message ----> Job 9: Subtask create_mysql_table [2020-08-04 21:22:59,637] {{dagbag.py:90}} INFO - Filling up the DagBag from /usr/local/airflow/dags/store_DAG.py
time---> [2020-08-04 21:22:59,773] filename and line ---> {{base_task_runner.py:115}} INFO message ----> Job 9: Subtask create_mysql_table [2020-08-04 21:22:59,772] {{cli.py:516}} INFO - Running <TaskInstance: store_DAG.create_mysql_table 2020-08-04T21:22:31.721187+00:00 [running]> on host efcc5644348e
time---> [2020-08-04 21:22:59,814] filename and line ---> {{mysql_operator.py:61}} INFO message ----> Executing: CREATE TABLE IF NOT EXISTS clean_store_transactions(STORE_ID varchar(50), STORE_LOCATION varchar(50), PRODUCT_CATEGORY varchar(50), PRODUCT_ID int, MRP float, CP float, DISCOUNT float, SP float, DATE date);
time---> [2020-08-04 21:22:59,839] filename and line ---> {{taskinstance.py:1047}} ERROR message ----> The conn_id `mysql_conn` isn't defined
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 922, in _run_raw_task
    result = task_copy.execute(context=context)
  File "/usr/local/lib/python3.7/site-packages/airflow/operators/mysql_operator.py", line 67, in execute
    parameters=self.parameters)
  File "/usr/local/lib/python3.7/site-packages/airflow/hooks/dbapi_hook.py", line 159, in run
    with closing(self.get_conn()) as conn:
  File "/usr/local/lib/python3.7/site-packages/airflow/hooks/mysql_hook.py", line 72, in get_conn
    conn = self.get_connection(self.mysql_conn_id)
  File "/usr/local/lib/python3.7/site-packages/airflow/hooks/base_hook.py", line 81, in get_connection
    conn = random.choice(list(cls.get_connections(conn_id)))
  File "/usr/local/lib/python3.7/site-packages/airflow/hooks/base_hook.py", line 76, in get_connections
    conns = cls._get_connections_from_db(conn_id)
  File "/usr/local/lib/python3.7/site-packages/airflow/utils/db.py", line 74, in wrapper
    return func(*args, **kwargs)
  File "/usr/local/lib/python3.7/site-packages/airflow/hooks/base_hook.py", line 59, in _get_connections_from_db
    "The conn_id `{0}` isn't defined".format(conn_id))
airflow.exceptions.AirflowException: The conn_id `mysql_conn` isn't defined
time---> [2020-08-04 21:22:59,842] filename and line ---> {{taskinstance.py:1076}} INFO message ----> All retries failed; marking task as FAILED
time---> [2020-08-04 21:22:59,880] filename and line ---> {{base_task_runner.py:115}} INFO message ----> Job 9: Subtask create_mysql_table Traceback (most recent call last):
time---> [2020-08-04 21:22:59,881] filename and line ---> {{base_task_runner.py:115}} INFO message ----> Job 9: Subtask create_mysql_table   File "/usr/local/bin/airflow", line 32, in <module>
time---> [2020-08-04 21:22:59,881] filename and line ---> {{base_task_runner.py:115}} INFO message ----> Job 9: Subtask create_mysql_table     args.func(args)
time---> [2020-08-04 21:22:59,881] filename and line ---> {{base_task_runner.py:115}} INFO message ----> Job 9: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/utils/cli.py", line 74, in wrapper
time---> [2020-08-04 21:22:59,882] filename and line ---> {{base_task_runner.py:115}} INFO message ----> Job 9: Subtask create_mysql_table     return f(*args, **kwargs)
time---> [2020-08-04 21:22:59,882] filename and line ---> {{base_task_runner.py:115}} INFO message ----> Job 9: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/bin/cli.py", line 522, in run
time---> [2020-08-04 21:22:59,883] filename and line ---> {{base_task_runner.py:115}} INFO message ----> Job 9: Subtask create_mysql_table     _run(args, dag, ti)
time---> [2020-08-04 21:22:59,883] filename and line ---> {{base_task_runner.py:115}} INFO message ----> Job 9: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/bin/cli.py", line 440, in _run
time---> [2020-08-04 21:22:59,883] filename and line ---> {{base_task_runner.py:115}} INFO message ----> Job 9: Subtask create_mysql_table     pool=args.pool,
time---> [2020-08-04 21:22:59,884] filename and line ---> {{base_task_runner.py:115}} INFO message ----> Job 9: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/utils/db.py", line 74, in wrapper
time---> [2020-08-04 21:22:59,884] filename and line ---> {{base_task_runner.py:115}} INFO message ----> Job 9: Subtask create_mysql_table     return func(*args, **kwargs)
time---> [2020-08-04 21:22:59,885] filename and line ---> {{base_task_runner.py:115}} INFO message ----> Job 9: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/models/taskinstance.py", line 922, in _run_raw_task
time---> [2020-08-04 21:22:59,885] filename and line ---> {{base_task_runner.py:115}} INFO message ----> Job 9: Subtask create_mysql_table     result = task_copy.execute(context=context)
time---> [2020-08-04 21:22:59,885] filename and line ---> {{base_task_runner.py:115}} INFO message ----> Job 9: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/operators/mysql_operator.py", line 67, in execute
time---> [2020-08-04 21:22:59,886] filename and line ---> {{base_task_runner.py:115}} INFO message ----> Job 9: Subtask create_mysql_table     parameters=self.parameters)
time---> [2020-08-04 21:22:59,886] filename and line ---> {{base_task_runner.py:115}} INFO message ----> Job 9: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/hooks/dbapi_hook.py", line 159, in run
time---> [2020-08-04 21:22:59,886] filename and line ---> {{base_task_runner.py:115}} INFO message ----> Job 9: Subtask create_mysql_table     with closing(self.get_conn()) as conn:
time---> [2020-08-04 21:22:59,887] filename and line ---> {{base_task_runner.py:115}} INFO message ----> Job 9: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/hooks/mysql_hook.py", line 72, in get_conn
time---> [2020-08-04 21:22:59,902] filename and line ---> {{base_task_runner.py:115}} INFO message ----> Job 9: Subtask create_mysql_table     conn = self.get_connection(self.mysql_conn_id)
time---> [2020-08-04 21:22:59,904] filename and line ---> {{base_task_runner.py:115}} INFO message ----> Job 9: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/hooks/base_hook.py", line 81, in get_connection
time---> [2020-08-04 21:22:59,906] filename and line ---> {{base_task_runner.py:115}} INFO message ----> Job 9: Subtask create_mysql_table     conn = random.choice(list(cls.get_connections(conn_id)))
time---> [2020-08-04 21:22:59,906] filename and line ---> {{base_task_runner.py:115}} INFO message ----> Job 9: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/hooks/base_hook.py", line 76, in get_connections
time---> [2020-08-04 21:22:59,907] filename and line ---> {{base_task_runner.py:115}} INFO message ----> Job 9: Subtask create_mysql_table     conns = cls._get_connections_from_db(conn_id)
time---> [2020-08-04 21:22:59,908] filename and line ---> {{base_task_runner.py:115}} INFO message ----> Job 9: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/utils/db.py", line 74, in wrapper
time---> [2020-08-04 21:22:59,908] filename and line ---> {{base_task_runner.py:115}} INFO message ----> Job 9: Subtask create_mysql_table     return func(*args, **kwargs)
time---> [2020-08-04 21:22:59,909] filename and line ---> {{base_task_runner.py:115}} INFO message ----> Job 9: Subtask create_mysql_table   File "/usr/local/lib/python3.7/site-packages/airflow/hooks/base_hook.py", line 59, in _get_connections_from_db
time---> [2020-08-04 21:22:59,909] filename and line ---> {{base_task_runner.py:115}} INFO message ----> Job 9: Subtask create_mysql_table     "The conn_id `{0}` isn't defined".format(conn_id))
time---> [2020-08-04 21:22:59,910] filename and line ---> {{base_task_runner.py:115}} INFO message ----> Job 9: Subtask create_mysql_table airflow.exceptions.AirflowException: The conn_id `mysql_conn` isn't defined
time---> [2020-08-04 21:23:01,931] filename and line ---> {{logging_mixin.py:95}} INFO message ----> [[34m2020-08-04 21:23:01,931[0m] {{[34mlocal_task_job.py:[0m105}} INFO[0m - Task exited with return code 1[0m
